Update Contributing.md   corrected wrong words. (#4537)
OfficeDev/office-js-docs-pr: Contributing.md

        cd <repo name>
        git remote add upstream https://github.com/OfficeDev/<repo name>.git

>         fetch upstream

Reviewer: Shouldn't this still say "git"?

```suggestion
        git fetch upstream
```

3. Submit a pull request to the main repository.
4. Delete the branch.

> **Important** Limit each branch to a single concept/article to streamline the workflow and reduce the chance of merge conflicts. Content appropriate for a new branch includes:

Reviewer: ```suggestion
**Important**: Limit each branch to a single concept/article to streamline the workflow and reduce the chance of merge conflicts. Content appropriate for a new branch includes:
```

One of the site administrators will process your pull request. Your pull request will surface on the OfficeDev/<repo name> site under Issues. When the pull request is accepted, the issue will be resolved.

#### Create a new branch after the merge

After a branch is successfully merged (that is, your pull request is accepted), don't continue working in that local branch. This can lead to merge conflicts if you submit another pull request. To do another update, create a new local branch from the successfully merged upstream branch, and then delete your initial local branch.

> For example, if your local branch X was successfully merged into the OfficeDev/Microsoft-graph-docs master branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/Microsoft-graph-docs master branch. To do this, open GitBash and execute the following commands.

Reviewer: ```suggestion
For example, if your local branch X was successfully merged into the OfficeDev/office-js-docs-pr main branch and you want to make additional updates to the content that was merged. Create a new local branch, X2, from the OfficeDev/office-js-docs-pr main branch. To do this, open Git Bash and execute the following commands.
```

>     cd Microsoft-graph-docs

Reviewer: ```suggestion
    cd office-js-docs-pr
```

>     git pull upstream master: X2

Reviewer: ```suggestion
    git pull upstream main:X2
```

1. In the Git Bash command prompt, type `git checkout master`. This ensures that you aren't in the branch to be deleted (which isn't allowed).
2. Next, at the command prompt, type `git branch -d <branch name>`. This deletes the branch on your computer only if it has been successfully merged to the upstream repository. (You can override this behavior with the `–D` flag, but first be sure you want to do this.)

> 3. Finally, type `git push origin:<branch name>` at the command prompt (a space before the colon and no space after it).  This will delete the branch on your GitHub fork.  

Reviewer: ```suggestion
3. Finally, type `git push origin :<branch name>` at the command prompt (a space before the colon and no space after it).  This will delete the branch on your GitHub fork.  
```
